src/Serilog.Sinks.Raygun/Sinks/Raygun/RaygunSink.cs

```diff
@@ -109,12 +113,37 @@ public void Emit(LogEvent logEvent)
                 : new RaygunErrorMessage()
                 {
                     ClassName = logEvent.MessageTemplate.Text,
-                    Message = logEvent.MessageTemplate.Text,     
+                    Message = logEvent.MessageTemplate.Text,
                     Data = logEvent.Properties.ToDictionary(k => k.Key, v => v.Value.ToString())
                 };
 
             // Add user when requested
-            if (!string.IsNullOrWhiteSpace(_userNameProperty) &&
+            if (!string.IsNullOrWhiteSpace(_userInfoProperty) &&
+                logEvent.Properties.ContainsKey(_userInfoProperty) &&
+                logEvent.Properties[_userInfoProperty] != null)
```

Comment:
This could be written as `logEvent.Properties.TryGetValue(_userInfoProperty, out var userInfoProperty)` to save two additional dictionary lookups below"

Reply:
Good idea, I have made this change :+1: